### ISSUE TITLE
Error when using custom output format with "{source_abs}" placeholder

### ISSUE THREAD

**Issue by apallier** [MarketSquare/robotframework-robocop#1621]
## Environment:

Robocop version: 7.2.0
OS: Ubuntu 24.04.1
Python version: 3.12.3

## Steps to reproduce:

Run robocop check on any Robot Framework file having an issue:

```shell
robocop check --configure print_issues.issue_format="{source_abs}" test.robot
```

## Actual Behavior:
The command fails with the following error:

`KeyError: 'source_abs'`

## Expected Behavior:
The command should execute successfully and include the absolute source path in the output as specified by the `{source_abs}` format placeholder.

The documentation indicates it should work to for the "extended" output type => https://robocop.dev/stable/linter/reports/print_issues/#issue-format

## Remarks

- The {source_abs} placeholder is documented/expected to be available for formatting issue output. See https://robocop.dev/stable/configuration/configuration_reference/#issue-format

- **Workaround**: the following command works for the "simple" output type:

```shell
robocop check --issue-format "{source_abs}" --configure print_issues.output_format=simple test.robot
```

**Comment by bhirsz** [comment]
Fixed - will be released in 8.0 .
